modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -580,6 +580,8 @@
 
     <string name="filters">Filters</string>
     <string name="episode_filters">Episode Filters</string>
+    <string name="no_results_for_filters">No results found for filter %s</string>
```

Comment:
Can we reuse the "filters_no_episodes" string used for the phone app? Since the filter title is already shown at the top, we can skip adding it here again.